.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Used to report bugs
+title: ''
+labels: problem/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**I tried this**
+Steps the caused the bug/error.
+
+**I expected this**
+A clear and concise description of what you expected to happen.
+
+**Instead this happened**
+A clear and concise description of what the unexpected outcome was.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Kubectl Version: [e.g. 22]
```

Comment:
I wonder if we should just ask for the output of `kubectl version` here.

Reply:
Yeah, I agree. I changed this to [output from `kubectl version`]